plugins/woocommerce/src/Admin/API/Reports/Customers/Controller.php

```diff
@@ -523,6 +524,22 @@ public function get_collection_params() {
 				'type' => 'integer',
 			),
 		);
+		$params['filter_empty']            = array(
+			'description'       => __( 'Filter out results where all passed fields are empty', 'woocommerce' ),
```

Comment:
From this description, it sounds like _every_ field name passed here would have to be empty in the same record before it would get skipped. But when trying this out, it actually seems to be if _any_ of the passed field names are empty.

For example, if I have a customer record that has an empty email field, but has `US` in the country field, and I do `filter_empty=email,country`, the record is still skipped, even though it has a country value.

I could be misinterpreting the intention, though. Is this supposed to be "all passed fields" or "any passed field"?

Reply:
> For example, if I have a customer record that has an empty email field, but has US in the country field, and I do filter_empty=email,country, the record is still skipped, even though it has a country value.

Yeah, that's the intention. We want to make sure that passed fields won't be empty in the given result

I changed the description to

> Filter out results where any of the passed fields are empty

How does this sound?

Reply:
Yep, that's more clear 👍